[develop] Updating Develop documentation with applicable changes from v3.0.0 release (#1281)
ufs-community/ufs-srweather-app: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst

   Setting ``RUN_ENVIR`` to "community" is recommended in most cases for users who are not running in NCO's production environment. Valid values: ``"nco"`` | ``"community"``

``MACHINE``: (Default: "BIG_COMPUTER")

>    The machine (a.k.a. platform or system) on which the workflow will run. Currently supported platforms are listed on the :srw-wiki:`SRW App Wiki page <Supported-Platforms-and-Compilers>`. When running the SRW App on any ParallelWorks/NOAA Cloud system, use "NOAACLOUD" regardless of the underlying system (AWS, GCP, or Azure). Valid values: ``"HERA"`` | ``"ORION"`` | ``"HERCULES"`` | ``"DERECHO"`` | `"GAEA-C6"`` |  ``"NOAACLOUD"`` | ``"SINGULARITY"`` | ` (Check ``ufs-srweather-app/ush/valid_param_vals.yaml`` for the most up-to-date list of supported platforms.)

Reviewer: ```suggestion
   The machine (a.k.a. platform or system) on which the workflow will run. Currently supported platforms are listed on the :srw-wiki:`SRW App Wiki page <Supported-Platforms-and-Compilers>`. When running the SRW App on any ParallelWorks/NOAA Cloud system, use "NOAACLOUD" regardless of the underlying system (AWS, GCP, or Azure). Valid values: ``"HERA"`` | ``"ORION"`` | ``"HERCULES"`` | ``"DERECHO"`` | ``"GAEA-C6"`` |  ``"NOAACLOUD"`` | ``"SINGULARITY"`` | ` (Check ``ufs-srweather-app/ush/experiment.jsonschema`` for the most up-to-date list of platforms where the SRW App can run.)
```

>    The machine (a.k.a. platform or system) on which the workflow will run. Currently supported platforms are listed on the :srw-wiki:`SRW App Wiki page <Supported-Platforms-and-Compilers>`. When running the SRW App on any ParallelWorks/NOAA Cloud system, use "NOAACLOUD" regardless of the underlying system (AWS, GCP, or Azure). Valid values: ``"HERA"`` | ``"ORION"`` | ``"HERCULES"`` | ``"JET"`` | ``"DERECHO"`` | ``"GAEA"`` | ``"GAEA-C6"`` |  ``"NOAACLOUD"`` | ``"SINGULARITY"`` | ` (Check ``ufs-srweather-app/ush/valid_param_vals.yaml`` for the most up-to-date list of supported platforms.)

Reviewer: @jdkublnick I realize it's not release-related, but you should probably remove JET and GAEA C5 here, since we're not supporting those systems anymore. 
```suggestion
   The machine (a.k.a. platform or system) on which the workflow will run. Currently supported platforms are listed on the :srw-wiki:`SRW App Wiki page <Supported-Platforms-and-Compilers>`. When running the SRW App on any ParallelWorks/NOAA Cloud system, use "NOAACLOUD" regardless of the underlying system (AWS, GCP, or Azure). Valid values: ``"HERA"`` | ``"ORION"`` | ``"HERCULES"`` | ``"DERECHO"`` | `"GAEA-C6"`` |  ``"NOAACLOUD"`` | ``"SINGULARITY"`` | ` (Check ``ufs-srweather-app/ush/valid_param_vals.yaml`` for the most up-to-date list of supported platforms.)
```

   | ``"FV3_HRRR"``
   | ``"FV3_WoFS_v0"``
   | ``"FV3_RAP"``
     ``"SUBCONUS_CO_3km"``

>      ``SUBCONUS_CO_1km"``

Reviewer: ```suggestion
     ``"SUBCONUS_CO_1km"``
```

>    Other valid values can be found in the ``ush/valid_param_vals.yaml`` `file <https://github.com/ufs-community/ufs-srweather-app/blob/release/public-v3.0.0/ush/valid_param_vals.yaml>`__, but users cannot expect full support for these schemes.

Reviewer: The problem here is that `valid_param_vals.yaml` just doesn't exist in the `develop` branch anymore. We need to point at wherever valid parameters are specified. @MichaelLueken can you suggest something?

Reviewer 2: There are now two files that take care of the valid parameters - `ush/experiment.jsonschema` and `ush/user.jsonschema`.

The `ush/user.jsonschema` file is checking against the machine that the experiments are being generated on, while `ush/experiment.jsonschema` is checking the rest of the parameters for a given experiment.

Both files take the role of the old `valid_param_vals.yaml` file.

   | ``"FV3_WoFS_v0"``
   | ``"FV3_RAP"``

>    Other valid parameter values are now defined in the ``ush/experiment.jsonschema`` `file <https://github.com/ufs-community/ufs-srweather-app/blob/develop/ush/experiment.jsonschema>`__ and the ``ush/user.jsonschema`` `file <https://github.com/ufs-community/ufs-srweather-app/blob/develop/ush/user.jsonschema>`__. The ``user.jsonschema`` file validates parameters specific to the machine generating the experiment, while ``experiment.jsonschema`` checks all other experiment parameters. Together, they replace the previous ``ush/valid_param_vals.yaml`` file.

Reviewer: The full list of _supported_ physics suites (to replace the one above) is: 
FV3_GFS_v16
FV3_HRRR
FV3_HRRR_gf
FV3_WoFS_v0
FV3_RAP
RRFS_sas
